src/pages/page-not-found/page-not-found.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import styles from './page-not-found.module.css';
+
+export const PageNotFound: React.FC = () => {
+  return (
+    <body>
+      <div className={styles.page_not_found}>
+        <h1 className={styles.h1}>404</h1>
```

Comment:
Did you check to see if the styling would still be applied to the h1 and h3 tags if you didn't add the classnames to them?

Reply:
They appear to be necessary